ccmlib/scylla_repository.py

```diff
@@ -204,7 +206,7 @@ def normalize_scylla_version(version):
         return version
 
     version = version.replace('-', '~').replace('.rc', '~rc')
-    version = version if re.match(r'\d*.\d*.0~rc', version) else version.replace('~rc', '.0~rc')
```

Comment:
Do we expectect `.0~` always in the version. Can it be `.1~` for example?

Reply:
no, release candidate are always on the first release, 2023.1.0~rc0. 5.2.0~rc0